### DISCUSSION TITLE
Do react-native-web versions track react-native versions?

### DISCUSSION THREAD

**Discussion by bgourlie** [necolas/react-native-web#2398] [Q&A] [answered]
I see react native web has versions 0.16, 0.17, 0.18... Do these track with major versions of react? For example, if I'm using version 16 of react/react-dom, do I need react-native-web 0.16?

**Comment by necolas** [comment, marked as answer]
No, there is no relationship between the version numbers of `react-native-web` and those of `react` or `react-native`. The peer dependency on `react` and `react-dom` is specified in the `package.json` of every `react-native-web` release. And the "works best with..." version of `react-native` to use with a given release of `react-native-web` is usually included in the README
